politeiad/backendv2/tstorebe/tstore/record.go

```diff
@@ -800,35 +800,35 @@ func getSortedKeys(blobs map[string][]byte) []string {
 }
 
 // Record returns the specified version of the record.
-func (t *Tstore) Record(token []byte, version uint32) (*backend.Record, error) {
+func (t *TstoreClient) Record(token []byte, version uint32) (*backend.Record, error) {
```

Comment:
I don't think anything in this file needs to be changed.  For example, `tstoreClient.Record` would just be:

```
 func (t *tstoreClient) Record(token []byte, version uint32) (*backend.Record, error) {
	 return t.tlog.Record(token, version)
 }
```